packages/js/data/src/tax-classes/index.ts

```diff
@@ -4,20 +4,20 @@
 import { createCrudDataStore } from '../crud';
 import { STORE_NAME, WC_TAX_CLASSES_NAMESPACE } from './constants';
 import * as resolvers from './resolvers';
-import { TaxClass, TaxClassActions, TaxClassSelectors } from './types';
+import { TaxClassActions, TaxClassSelectors } from './types';
 
-export const store = createCrudDataStore<
-	TaxClass,
-	TaxClassActions,
-	TaxClassSelectors
->( {
-	storeName: STORE_NAME,
-	resourceName: 'TaxClass',
-	pluralResourceName: 'TaxClasses',
-	namespace: WC_TAX_CLASSES_NAMESPACE,
-	storeConfig: {
-		resolvers,
-	},
-} );
+export const store = createCrudDataStore< TaxClassActions, TaxClassSelectors >(
+	{
+		storeName: STORE_NAME,
+		resourceName: 'TaxClass',
+		pluralResourceName: 'TaxClasses',
+		namespace: WC_TAX_CLASSES_NAMESPACE,
+		storeConfig: {
+			resolvers: {
+				...resolvers,
+			},
```

Comment:
```suggestion
			resolvers,
```
How come the extra spreading?

Reply:
Fixed with cebfac7f615cbe00efd5aa86077b733748b373f6